### PR TITLE
changed font size on h2

### PR DESCRIPTION
**Pull request by haiderf7**
## Checklist before requesting a review

* [x] I created a new branch before starting with this ticket?
* [x] I commented the issue after finish in the Frontend Team Board?
* [x] I moved the card to Quality Assurance after finish?

## Did you run into any issues?

- No, issues encountered.
- Yes, I encountered issues. (Please describe below).

## Screenshots:

- No screenshots to include.
- Yes, the screenshots are included in the table below.

| Before | After |
---------|--------
|        |       |

## Feedback

- No, I don't want feedback.
- Yes, I want feedback.

### CODE REVIEW

**Review by runeunhjem — COMMENTED** [review]
Think you have imported it into the wrong index.js :)
<script defer src="/index.js" type="module"></script>

This means root/index.js

**Comment by haiderf7** [comment]
i think i have it in the right index.js now?

File: index.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ import { createListing } from './src/js/listeners/post/createListing.js';
 import { showListings } from './src/js/views/admin/showListings.js';
 import { searchListings, filterListings } from './src/js/listeners/post/searchListing.js'
 import '/src/scss/index.scss';
+import { adjustH2FontSize } from './listingTemplate.js';
```
Comment (Olekrr) [inline index.js:16]: Your file import here seems to be incorrect and is resulting in a linting error. Please take a look at this

**Review by Olekrr — CHANGES_REQUESTED** [review]

**Comment by haiderf7** [comment]
fixed it now

**Comment by Olekrr** [comment]
This issue has been solved elsewhere, as such this PR will be closed without merging.

**Pull request closed without merging**